components/CreateObject.coffee

```diff
@@ -1,23 +1,19 @@
 noflo = require 'noflo'
 
-class CreateObject extends noflo.Component
-  constructor: ->
-    @inPorts = new noflo.InPorts
-      start:
-        datatype: 'bang'
-        description: 'Signal to create a new object'
-    @outPorts = new noflo.OutPorts
-      out:
-        datatype: 'object'
-        description: 'A new empty object'
+exports.getComponent = ->
+  c = new noflo.Component
 
```

Comment:
As part of noflo/noflo#410 it would be good to make sure all components have meaningful descriptions, and icons if available